modules/enabling_encapsulation.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/optimizing-cpu-usage.adoc
+
+[id="enabling-encapsulation_{context}"]
+= Working with mount namespace encapsulation
+
+As an OpenShift system administrator, you want `systemd` to consume less resources so that you can run more workloads on my system. You want to enable and disable this setting, and verify whether encapsulation is in effect.
+
+== Enabling and disabling encapsulation
+
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
+
+* You have installed the OpenShift CLI (oc).
+
+.Procedure
+
+. Add the following YAML to the `MachineConfig.yaml` file to control whether encapsulation is enabled or disabled:
```

Comment:
This procedure feels incomplete - Just creating a yaml file won't actually change the cluster.  Do we want to copy or reference the procedure from https://docs.openshift.com/container-platform/4.11/post_installation_configuration/machine-configuration-tasks.html#installation-special-config-chrony_post-install-machine-configuration-tasks and just use that as a template for "how to add a new MachineConfig object"?

Reply:
Or at least, if we're going the non-butane route, we should add a step that asks the user to run `oc apply -f MachineConfig.yaml` to apply the change, and then monitor the MCP pools (as shown in https://docs.openshift.com/container-platform/4.11/post_installation_configuration/machine-configuration-tasks.html#checking-mco-status_post-install-machine-configuration-tasks) before verifying the feature is active.

Reply:
Going with the non-butane route: line 60 
$ oc create -f mount_namespace_config.yaml

Reply:
Done.